classroom.rb

```diff
@@ -0,0 +1,51 @@
+require_relative './student'
+require_relative './person'
+require_relative './classroom'
+
+class Classroom
+  def initialize(label)
+    @label = label
+  end
+  attr_accessor :label
+
+  def add_student
+    new.Student(@label)
+  end
+end
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- [ ] **Required:** The class `Book` should be initialized with a rental array. eg. `@rental = []`